pterodactyl.php

```diff
@@ -1208,11 +1240,36 @@ public function getAdminAddFields($package, $vars = null)
         $this->loadLib('pterodactyl_service');
         $service_helper = new PterodactylService();
 
+        // Get configurable options
+        if (!isset($this->Record)) {
+            Loader::loadComponents($this, ['Record']);
+        }
+        if (!isset($this->Form)) {
+            Loader::loadHelpers($this, ['Form']);
+        }
+
+        $configurable_options = $this->Form->collapseObjectArray(
+            $this->Record->select(['package_options.id', 'package_options.name'])
+                ->from('package_options')
+                ->where('package_options.company_id', '=', Configure::get('Blesta.company_id'))
+                ->where('package_options.id', 'IN', array_keys($vars->configoptions))
```

Comment:
I get this error when visiting the admin service add page:
Undefined property: stdClass::$configoptions on line 1255 in ...\blesta-5.0\public_html\components\modules\pterodactyl\pterodactyl.php

Reply:
I updated this part